src/rest.cpp

```diff
@@ -3,6 +3,7 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <attributes.h>
 #include <chain.h>
 #include <chainparams.h>
 #include <core_io.h>
```

Comment:
should include https://github.com/bitcoin/bitcoin/commit/9cc0230cfc1ae9b9c1c905cd9ac613bc98bfa43a#diff-590507deaf686d6571f73979df5f3c6da6013a13e597f390b8facce39f7c69d1R10-R20

Reply:
I've covered header reordering in my comment [here](https://github.com/dashpay/dash/pull/4028#discussion_r596808835)

Reply:
We should try to keep backports as close as reasonably possible to the upstream commit. Please include these changes, as it doesn't hurt anything

Reply:
Resolved in eb3511395abd6fd0407534ba2b5468b090c3008b